packages/views/basics/autoSuggest/index.js

```diff
@@ -34,7 +36,7 @@ const AutoSuggest = ({ // eslint-disable-line max-statements
 
   const onKeyPressDownOrUp = (action) => {
     const rowHeight = 44;
-    const filteredItemsLength = getFilterList().length;
+    const filteredItemsLength = filterList().length;
```

Comment:
now "filterList" it isn't a function anymore
```
 const filteredItemsLength = filterList.length;
 ```

Reply:
@soroushm If it's implemented like that, then it leads to Deep Scan errors, particularly `Variable 'filterList' has a non-function value. But it is called as a function.`. The errors can be found in https://deepscan.io/dashboard/#view=project&tid=6759&pid=8871&bid=569682&subview=pull-request&prid=1187282

Reply:
@soroushm Already implemented